Makefile

```diff
@@ -95,6 +95,7 @@ terraform_gen:
     	-exec sed -i".bak" 's#github\.com/hashicorp/terraform/version#github.com/fugue/regula/v2/pkg/terraform/version#' '{}' \;
 	find pkg/terraform/ -name '*.bak' -delete
 	find pkg/terraform/ -name '*_test.go' -delete
+	./scripts/modify-terraform.sh
```

Comment:
Running `make terraform_gen` also produces this diff, which stops regula from compiling:

```diff
diff --git a/pkg/terraform/configs/configload/loader_snapshot.go b/pkg/terraform/configs/configload/loader_snapshot.go
index 6ca8b4f..243939f 100644
--- a/pkg/terraform/configs/configload/loader_snapshot.go
+++ b/pkg/terraform/configs/configload/loader_snapshot.go
@@ -325,10 +325,6 @@ func (fs snapshotFS) Chmod(name string, mode os.FileMode) error {
        return fmt.Errorf("cannot set file mode inside configuration snapshot")
 }

-func (fs snapshotFS) Chown(string, int, int) error {
-       return fmt.Errorf("cannot set file owner inside configuration snapshot")
-}
-
 func (fs snapshotFS) Chtimes(name string, atime, mtime time.Time) error {
        return fmt.Errorf("cannot set file times inside configuration snapshot")
 }

```

I'm not sure of the context this make target runs in - i.e. if it's ever orchestrated with further terraform modifications.

Reply:
I've added this to the patch